sentry-test-support/src/main/kotlin/io/sentry/test/assertions.kt

```diff
@@ -10,7 +10,7 @@ import io.sentry.SentryOptions
 /**
  * Verifies is [SentryEnvelope] contains first event matching a predicate.
  */
-inline fun checkEvent(noinline predicate: (SentryEvent) -> Unit): SentryEnvelope {
+fun checkEvent(predicate: (SentryEvent) -> Unit): SentryEnvelope {
```

Comment:
no reason to be inlined, first because has no effect for this usecase, 2nd because its a testing class